src/trace-writer.ts

```diff
@@ -145,14 +145,14 @@ export class TraceWriter extends common.Service {
             'node ' + pjson.name + ' v' + pjson.version;
         labels[TraceLabels.GCE_HOSTNAME] = hostname;
         if (instanceId) {
-          labels[TraceLabels.GCE_INSTANCE_ID] = instanceId;
+          labels[TraceLabels.GCE_INSTANCE_ID] = `${instanceId}`;
```

Comment:
I'd rather define a local function that adds a new label to `labels`, which always converts the given value to a string. That'll make this code more extensible in the future when adding new labels.